pkgs/by-name/ed/edusong/package.nix

```
@@ -6,25 +6,23 @@
 
 stdenvNoCC.mkDerivation rec {
   pname = "edusong";
-  version = "1.0"; # The upstream doesn't provide the version
+  version = "4.0";
 
   src = fetchzip {
     name = "${pname}-${version}";
     url = "https://language.moe.gov.tw/001/Upload/Files/site_content/M0001/eduSong_Unicode.zip";
-    hash = "sha256-pIG1EbFGf2O2AzM4+HCCvGPodBBwUt7ozpb+BpPk5Kw=";
+    hash = "sha256-4NBnwMrYufeZbgSiD2fAhe4tuy0aAA5u9tWwjQQjEQk=";
   };
 
   installPhase = ''
     mkdir -p $out/share/fonts/
-    mv *.ttf $out/share/fonts/
+    mv eduSong_Unicode*.ttf $out/share/fonts/eduSong_Unicode\(2024年12月\).ttf
   '';
 
   meta = {
-    description = "The MOE Standard Song Font, a Chinese font by the Ministry of Education, ROC (Taiwan)";
+    description = "The MOE Song font, a Song-style Chinese character typeface";
```

Current version of [`pkg/README.md`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes) forbids `meta.description` starting with an article.

```suggestion
    description = "MOE Song font, a Song-style Chinese character typeface";
```